docs/services/motion/_index.md

```diff
@@ -398,10 +400,11 @@ import (
   "go.viam.com/rdk/services/motion"
 )
 
-// < Insert code to connect to your robot
+// < Insert code to connect to your robot >
```

Comment:
I think get rid of the angular brackets here and on the next line. I think it's weird in comments - or just get rid of it on the next line